templates/travis/.travis/script.sh.j2

```diff
@@ -40,12 +41,15 @@ fi
 
 ./generate.sh pulpcore python
 pip install ./pulpcore-client
-{% if plugin_name == 'pulpcore' %}  ./generate.sh pulp_file python
+{%- if plugin_name == 'pulpcore' %}
+  ./generate.sh pulp_file python
   pip install ./pulp_file-client
   ./generate.sh pulp_certguard python
-  pip install ./pulp_certguard-client{% else %}  ./generate.sh {{ plugin_snake }} python
+  pip install ./pulp_certguard-client
+{%- else %}
+  ./generate.sh {{ plugin_snake }} python
   pip install ./{{ plugin_snake }}-client
-{% endif %}
+{%- endif %}
```

Comment:
Thank you for this cleanup!

Reply:
you are welcome. (i dislike trailing WS...)